info.olasagasti.revelation.json

```diff
@@ -0,0 +1,61 @@
+{
+    "app-id": "info.olasagasti.revelation",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.36",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "revelation",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=xdg-run/gvfs",
+        "--filesystem=home",
+        "--talk-name=org.gtk.vfs",
+        "--talk-name=org.gtk.vfs.*",
+        "--talk-name=org.gnome.ScreenSaver"
+    ],
+    "modules" : [
+        {
+            "name" : "libpwquality",
+            "buildsystem" : "autotools",
+            "config-opts": [
+                "--with-python-binary=python3"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://github.com/libpwquality/libpwquality/releases/download/libpwquality-1.4.2/libpwquality-1.4.2.tar.bz2",
+                    "sha256" : "5263e09ee62269c092f790ac159112aed3e66826a795e3afec85fdeac4281c8e"
+                }
+            ]
+        },
+        {
+            "name" : "pycryptodomex",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --no-index --no-build-isolation --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} pycryptodomex"
+            ],
+            "sources" : [
+                {
+                    "type" : "file",
+                    "url" : "https://files.pythonhosted.org/packages/f5/79/9d9206688385d1e7a5ff925e7aab1d685636256e34a409037aa7adbbe611/pycryptodomex-3.9.8.tar.gz",
+                    "sha256" : "48cc2cfc251f04a6142badeb666d1ff49ca6fdfc303fd72579f62b768aaa52b9"
+                }
+            ]
+        },
+        {
+            "name" : "revelation",
+            "buildsystem" : "autotools",
+            "config-opts": [
+                "--disable-mime-update"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/mikelolasagasti/revelation.git",
+                    "branch" : "flathub-change"
```

Comment:
This needs to be a tagged release or a specific commit for builds reproducibility.

Reply:
I was using a branch to test changes required to build the flatpak on flathub that will merged and released. I was planning to do it after the review was complete, would it be mandatory to do it before/now?